MAUI/LinearGauge/animation.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Animation in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing animation of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Animation in .NET MAUI Linear Gauge (SfLinearGauge)
+
+All Linear Gauge elements such as axis along with ticks and labels, range, bar pointer, shape marker pointer and content marker pointer can be animated separately. 
+
+## Animate axis
+
+The [`EnableAxisAnimation`]() and [`AnimationDuration`]() properties in [`SfLinearGauge`]() is used to animate the scale along with the ticks and labels. The scale will have a fade-in with opacity animation when this [`EnableAxisAnimation`]() is set to true. By default, the [`EnableAxisAnimation`]() is set to false. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge EnableAxisAnimation="True" />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.EnableAxisAnimation = true;
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Animate axis in linear gauge](images/animation/animation_axis.gif)
+
+## Animate range
+
+The [`EnableRangeAnimation`]() and [`AnimationDuration`]() properties in [`SfLinearGauge`]() is used to  animate the scale along with the ticks and labels. The range will be have a fade-in with opacity animation when this [`EnableRangeAnimation`]() is set to true. By default, the [`EnableRangeAnimation`]() is set to false. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge EnableRangeAnimation="True" />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.EnableRangeAnimation = true;
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Animate range in linear gauge](images/animation/animation_range.gif)
+
+## Pointer animation
+
+The animation behavior is common for all the three pointers in Linear Gauge - shape, content and bar pointer. 
+
+All the above three pointers have the below properties for animation. 
+
+*  [`EnableAnimation`]() - Enable or disable the animation for pointer. The default value is `false`.
+*  [`AnimationDuration`]() - Sets the animation duration. The default value is 1000.
+*  [`AnimationEasing`]() - Sets the animation type. 
+
+The [`AnimationEasing`]() supports the below animations. The default animation type is `Easing.Linear`.
```

Comment:
The [`AnimationEasing`]() supports the following animations. The default animation type is `Easing.Linear`.